prototype2/multigrid/mgmesytec/DataParserTest.cpp

```diff
@@ -10,7 +10,11 @@ static Producer producer {"noserver", "nostream"};
 class MesytecDataTest : public TestBase {
 protected:
   NMXHists hists;
-  MesytecData mesytec{0, "nofile", 1}; // Dont dumptofile select module with 20 depth in z
+#ifdef DUMPTOFILE
+MesytecData mesytec{0, "nofile", 1}; // Dont dumptofile select module with 20 depth in z
```

Comment:
This is one of the solutions I suggested to Morten. He was slightly skeptical but seeing as you have implemented it, I say we use it.